CMakeLists.txt

```diff
@@ -3,7 +3,7 @@
 # GNU Lesser General Public License v2.1 or any later version.
 
 cmake_minimum_required(VERSION 3.12)
-project(GymIgnition VERSION 0.1)
+project(ScenarI/O VERSION 0.1)
```

Comment:
Nitpick: I personally prefer that the name used in the `project` call of CMake matches the one used in the export CMake configuration files, so that it clearly defines the CMake **package name** . In this context, then I would use `Scenario` . Furthermore, putting `/` in anything is basically a russian roulette of things that could go wrong.

Reply:
When I did it, I googled if this was allowed and what could go wrong, without finding any evidence (but I didn't searched extensively). Keeping the same name of the export makes sense, I'll change that, thanks!

Reply:
Done in https://github.com/robotology/gym-ignition/compare/877b753b1300d4ee2bffb67a338fa4399b972517..7b99f6b70e4686df0843e666392f4b248b970741